bindata/etcd/etcd-member-remove.sh

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 
+### Created by cluster-etcd-operator. DO NOT edit.
```

Comment:
maybe in the future, we should have the process add this string but not something we need today.